codegen/jennies/gotypes.go

```diff
@@ -127,7 +127,7 @@ func (g *GoTypes) generateFilesAtDepth(v cue.Value, kv *codegen.KindVersion, cur
 			PackageName:                    packageName,
 			Name:                           exportField(strings.Join(fieldName, "")),
 			NamePrefix:                     namePrefix,
-			AddKubernetesOpenAPIGenComment: g.AddKubernetesCodegen && !(len(fieldName) == 1 && fieldName[0] == "metadata"),
+			AddKubernetesOpenAPIGenComment: g.AddKubernetesCodegen && (len(fieldName) != 1 || fieldName[0] == "metadata"),
```

Comment:
This should be translated to 
```
g.AddKubernetesCodegen && (len(fieldName) != 1 || fieldName[0] != "metadata")
```
(missing the inversion on the second equality)

Reply:
Addressed [be607ff](https://github.com/grafana/grafana-app-sdk/pull/1025/commits/be607ffd062824430569b35edb8871b45a84dd49)

Reply:
Awesome, thank you! I missed the failing test in the PR test action.